src/main/kotlin/dev/restate/sdktesting/tests/Cancellation.kt

```diff
@@ -123,7 +123,7 @@ class Cancellation {
     await.ignoreException(TimeoutCancellationException::class.java).until {
       runBlocking {
         testUtilsClient.cancelInvocation(id, idempotentCallOptions)
-        withTimeout(1.seconds) { cancelTestClient.verifyTest() }
+        withTimeout(1.seconds) { cancelTestClient.verifyTest(idempotentCallOptions) }
```

Comment:
why u need idempotent call options here? this is in an await loop

Reply:
Which repeats on `TimeoutCancellationException` but fails on any other exception.

Reply:
ok